.travis.yml

```diff
@@ -1,36 +1,63 @@
 language: node_js
 matrix:
   fast_finish: true
-  # Generally using active LTS versions here, see https://github.com/nodejs/Release
+  # Generally using active LTS versions here, see https://github.com/nodejs/Release 
   include:
-    - name: "Test client"
-      node_js: 12
-      env: RUN="npm run test:client"
-    - name: "Test server"
-      node_js: 12
-      env: RUN="npm run test:server"
-    - name: "Lint and try building"
-      node_js: 12
-      env: RUN="npx concurrently --kill-others-on-fail 'npm:lint' 'npm:build'"
+  - name: Test client
+    node_js: 12
+    env: RUN="npm run test:client"
+  - name: Test server
+    node_js: 12
+    env: RUN="npm run test:server"
+  - name: Lint and build
```

Comment:
Now that build+deploy might take longer, we could split the build to another container to increase parallelism. Not in this PR and only if it makes things faster of course. Just noting down.

Reply:
The deploy part uses the output of the build part, so they can't be parallelised.

In sequence it has to install npm deps, build the webpack stuff, bundle all up in a tarball, put the tarball on DigitalOcean Spaces, then run the deploy playbook.

The lint could be split out, but I doubt that would make much difference.

Reply:
Sounds good! 👍